src/CardContent/FileDetails/FileDetails.types.ts

```diff
@@ -9,6 +9,14 @@ export type FileDetailsProps = {
    * The download button text on search.
    */
   downloadButtonTextOnSearch?: string;
+  /**
+   * The download href link.
+   */
+  downloadLinkHref?: string;
```

Comment:
I think a lesson for next time we write a component like this is that nested props are a good idea. e.g. downloadLinkProps: { text, href, target, onClick} etc.

Too late to change now.